src/components/shared/AddRemoveForm.js

```diff
@@ -28,9 +28,9 @@ class AddRemoveForm extends React.Component {
   }
 
   getErrorMessage() {
-    const { errorMessage } = this.state
+    const { error } = this.state
 
-    return errorMessage ? <span className="help is-danger is-inline">{errorMessage}</span> : null
+    return error && error.message ? <span className="help is-danger is-inline">{error.message}</span> : null
```

Comment:
Actually found this error (no pun intended) while writing tests for it! I was setting an error object but trying to write it as a string, causing the error message not to show up.